apps/multiplatform/android/src/main/java/chat/simplex/app/views/helpers/Share.kt

```diff
@@ -96,7 +96,7 @@ fun imageMimeType(fileName: String): String {
 
 /** Before calling, make sure the user allows to write to external storage [Manifest.permission.WRITE_EXTERNAL_STORAGE] */
 fun saveImage(cxt: Context, ciFile: CIFile?) {
-  val filePath = getLoadedFilePath(cxt, ciFile)
+  val filePath = getLoadedFilePath(ciFile)
```

Comment:
why safeImage still needs cxt?

Reply:
also other functions in this file - they accept Context but don't pass it. Looks like there may be other places where Context is passed?

Reply:
done